openvdb_houdini/houdini/SOP_NodeVDB.cc

```diff
@@ -588,23 +587,42 @@ class SESIOpenVDBOpPolicy: public houdini_utils::OpPolicy
 };
 
 
-/// @brief OpPolicy for OpenVDB operator types at DWA
-class DWAOpenVDBOpPolicy: public houdini_utils::DWAOpPolicy
+/// @brief Default OpPolicy for OpenVDB operator types
```

Comment:
Shouldn't this say "SideFX OpPolicy for OpenVDB operator types"?

Reply:
Yes, well spotted, fixed.